python_modules/dagster/dagster/core/events/__init__.py

```diff
@@ -259,6 +266,16 @@ def log_resource_event(log_manager: DagsterLogManager, event: "DagsterEvent") ->
     log_manager.log_dagster_event(level=log_level, msg=event.message or "", dagster_event=event)
 
 
+def log_asset_materialization_planned_event(
+    log_manager: DagsterLogManager, event: "DagsterEvent"
+) -> None:
+    # asset_materialization_planned events have a log level "DEBUG" in order to hide these
+    # events by default in Dagit. Modifying filtering to select DEBUG events will show these events
+    # in Dagit run logs.
+    log_level = logging.DEBUG
+    log_manager.log_dagster_event(level=log_level, msg=event.message or "", dagster_event=event)
```

Comment:
no need to cargo-cult the pattern here and refactor when single use  - this can just be inlined

Reply:
✅